docs/Change_Log.md

```diff
@@ -1,5 +1,51 @@
 # Change Log
 
+#### May 3, 2023
+
+## Unique usernames on Discord
```

Comment:
```suggestion
## Unique usernames on Discord

#### May 3, 2023
```
Fix formatting in accordance with https://github.com/discord/discord-api-docs/pull/6162.